hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -587,6 +571,21 @@ msgid ""
 "contact Data Center Light Support."
 msgstr ""
 
+#~ msgid "How it works"
+#~ msgstr "So funktioniert es"
+
+#~ msgid "Your infrastructure"
+#~ msgstr "deine Infrastruktur"
+
+#~ msgid "Our inftrastructure"
+#~ msgstr "Unsere Infrastruktur"
+
+#~ msgid "Pricing"
+#~ msgstr "Preise"
```

Comment:
`584 #~ msgstr "Preise"` is a commented out line. Anyways, I will make this change for future reference.